src/core/agent/services/connectionService.test.ts

```diff
@@ -611,4 +626,84 @@ describe("Connection service of agent", () => {
       Agent.agent.signifyNotifications.addPendingOperationToQueue
     ).toBeCalledTimes(1);
   });
+
+  test("Can get connection History by id", async () => {
+    const connectionId = "connectionId";
+    const date1 = new Date("Sat Jul 27 2024 15:02:30 GMT+0700");
+    const date2 = new Date("Sat Jul 27 2024 15:12:04 GMT+0700");
```

Comment:
minor thing but can we make `date2` more recent than `date3` - just tests the sorting a bit better in case that regresses. So e.g. `15:45:04`